index.d.ts

```diff
@@ -150,16 +185,29 @@ interface MetricConfiguration<T extends string> {
 	name: string;
 	help: string;
 	labelNames?: T[] | readonly T[];
-	registers?: Registry[];
+	registers?: Registry<PrometheusContentType | OpenMetricsContentType>[];
```

Comment:
```suggestion
	registers?: (Registry<PrometheusContentType> | Registry<OpenMetricsContentType>)[];
```

right? A registry itself cannot contain either